tests/src/test/java/org/wso2/micro/gateway/tests/common/Constants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
shouldn't this be 2020?
```suggestion
 * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```